joerd/source/ned_base.py

```diff
@@ -19,122 +20,107 @@
 from osgeo import gdal
 import urllib2
 import shutil
+import yaml
+import time
 
 
-def __download_ned_file(img_name, zip_name, base_dir, ftp_server, base_path,
-                        options):
-    logger = logging.getLogger('ned')
-    output_file = os.path.join(base_dir, img_name)
+GLOBAL_CACHE = {}
```

Comment:
There are two instances of `NEDBase`, one for the normal NED and the other for topobathy. That said, it seems safer to keep the cache with the index since sharing it only saves us one call to `yaml.load`. Fixed in 10a5781.